server/app/routes.py

```diff
@@ -112,3 +111,158 @@ def delete_user(user_id):
         "message": "User deleted successfully.",
         "data": None
     }), 200
+
+
+# Routes:: Winnie
+
+# Routes for stores
+@app.route('/api/stores', methods=['GET'])
+def get_stores():
+    stores = Store.query.all()
+    stores_data = [store.to_dict() for store in stores]  # Serialize each Store object
+    return jsonify({
+        "status": "success",
+        "message": "success",
+        "data": stores_data
+    }), 200
+
+@app.route('/api/stores/<int:store_id>', methods=['GET'])
+def get_store(store_id):
+    store = Store.query.get(store_id)
+    if not store:
+        return jsonify({
+            "status": "Failed",
+            "message": "Store not found",
+            "data": None
+        }), 404
+    return jsonify({
+        "status": "success",
+        "message": "success",
+        "data": store.to_dict()
+    }), 201
+    
+
+@app.route('/api/stores', methods=['POST'])
+def create_store():
+    data = request.get_json()
+    if 'store_name' not in data or 'location' not in data:
+        return jsonify({
+            'status': 'Failed',
+            'message': 'Store_name and location fields are required',
+            'data': None
+        }), 400
+    
+    new_store = Store(store_name=data['store_name'], location=data['location'])
+    db.session.add(new_store)
+    db.session.commit()
+    return jsonify(new_store.to_dict()), 201
```

Comment:
Keep the response format consistent so that it maintains a consistent structure.
Use this as the reference:
```python
return jsonify({
            "status": "Failed",
            "message": "Email and user_id are required.",
            "data": None
        }), 400
```